src/instructlab/model/list.py

```diff
@@ -0,0 +1,138 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from pathlib import Path
+import logging
+import os
+import time
+
+# Third Party
+import click
+import pandas
```

Comment:
Panda is not a small library, it took a few seconds for me to load it. Are we ok adding that up to the CLI?

Reply:
removed, sorry!